### PR TITLE
add some copyright info

### PR DESCRIPTION
**Pull request by jianboqi**
we develop a mex file that can be used in Matlab. Now, CSF can handle Matlab marix and export ground points and non-ground points.

### CODE REVIEW

**Comment by cloudcompare** [comment]
It seems your branch has conflicts with the main one. You'll need to fix them first before I can merge the pull request.

**Comment by jianboqi** [comment]
how to fix the conflics?

File: plugins/qCSF/Rasterization.cpp [hunk 1]
```diff
@@ -129,7 +129,8 @@ bool Rasterization::RasterTerrain(Cloth& cloth, const wl::PointCloud& pc, std::v
 		//#pragma omp parallel for
 		for (int i = 0; i < cloth.getSize(); i++)
 		{
-			Particle &pcur = cloth.getParticleByIndex(i);
+			//Particle &pcur = cloth.getParticleByIndex(i);
+			Particle& pcur = cloth.getParticleByIndex(i);      //zwm
```
Comment (asmaloney) [inline plugins/qCSF/Rasterization.cpp]: No need to add unnecessary comments.

File: plugins/qCSF/PointCloud.h [hunk 2]
```diff
@@ -38,7 +38,7 @@ namespace wl
 			}
 
 			bbMin = bbMax = at(0);
-			for (size_t i = 1; i < size(); i++)
+			for (std::size_t i = 1; i < size(); i++)  //zwm
```
Comment (asmaloney) [inline plugins/qCSF/PointCloud.h]: No need to add unnecessary comments.

File: plugins/qCSF/Cloth.h [hunk 3]
```diff
@@ -78,7 +78,8 @@ class Cloth
 	inline Particle& getParticle(int x, int y) { return particles[y*num_particles_width + x]; }
 	inline const Particle& getParticle(int x, int y) const { return particles[y*num_particles_width + x]; }
 	
-	inline Particle getParticleByIndex(int index) { return particles[index]; }
+	//inline Particle getParticleByIndex(int index) { return particles[index]; }
+	inline Particle& getParticleByIndex(int index) { return particles[index]; }  //zwm
```
Comment (asmaloney) [inline plugins/qCSF/Cloth.h]: No need to add unnecessary comments.

**Comment by asmaloney** [comment]
There's a whole bunch of information about [resolving merge conflicts](https://stackoverflow.com/questions/161813/fix-merge-conflicts-in-git) online.  Exactly how you do it depends on the tools (editor/git interface/diff tool/merge tool) you are using.

Also it looks to me like there are multiple things going on in this PR: a change to the UI, fixes to copyright comments & changes for stdlib namespacing.  These should each be a separate pull request.

**Pull request closed without merging**